### PR TITLE
Feature/redux/articles

### PR DESCRIPTION
**Pull request by Tramsi**

### CODE REVIEW

File: src/components/NewsAndArticles/index.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { useSpring, animated } from 'react-spring';
 import useMedia from '../../Helpers/useMedia';
 import Article from '../Article';
 
-const ArticlesList = ({ articles, loading, error, animationDelay }) => {
+const ArticlesList = ({ data, loading, error, animationDelay }) => {
```
Comment (Nilegfx) [inline src/components/NewsAndArticles/index.jsx:16]: why did you change the prop name? wasn't it better to pass data as articles prop and keep the component props as is?

Comment (Tramsi) [reply src/components/NewsAndArticles/index.jsx:16]: I changed it to follow the convention to send data to components in "data" name and in component i can name it whatever I want
